docs/sources/flow/reference/components/discovery.docker.md

```diff
@@ -0,0 +1,167 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/discovery.docker
+title: discovery.docker
+---
+
+# discovery.docker
+
+`discovery.docker` discovers [Docker Engine][] containers and exposes them as targets.
+
+[Docker Engine]: https://docs.docker.com/engine/
+
+## Usage
+
+```river
+discovery.docker "LABEL" {
+  host = "DOCKER_ENGINE_HOST"
+}
+```
+
+## Arguments
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`host` | `string` | Address of the Docker Daemon to connect to. | | yes
```

Comment:
Thoughts on making this optional and reverting to either of the two defaults listed below in Linux/Windows depending on OS.

Reply:
The Windows default won't work out of the box unless you configure the Daemon a specific way (which also probably isn't recommended for security reasons since you have to disable TLS)